data/metadata.json

```diff
@@ -7,31 +7,63 @@
     }
   },
   "0x33f75ff0b830dcec": {
+    "features": {
+      "supported": ["self-custody", "mobile"]
```

Comment:
Should we maybe make a JSON file/registry of valid features & only allow features to be used that are in here?  This way we can avoid typos, keep track of all available wallet features, avoid bloating of new "features" being introduced by additional wallet listings, etc.

Also can display tooltip with what the feature actually means.

I'm thinking something along the lines of...

```json
[
    {
        "name": "self-custody",
        "description": "User manages their own key....."
    }
    ...
]
```

Reply:
Done